SemanticMetaTags.php

```diff
@@ -41,32 +33,15 @@ public static function load() {
 			include_once __DIR__ . '/vendor/autoload.php';
 		}
 
-		// In case extension.json is being used, the succeeding steps will
-		// be handled by the ExtensionRegistry
-		self::initExtension();
-
-		$GLOBALS['wgExtensionFunctions'][] = function() {
-			self::onExtensionFunction();
-		};
 	}
 
 	/**
 	 * @since 1.0
 	 */
 	public static function initExtension() {
```

Comment:
So basically this needs to be:

```
public static function initExtension( $credits = [] ) {
```

Done!
SRF does not provide the `$credits` parameter. This needs to be fixed.

Reply:
In this case, yes. The reason it didn't crashed is because we use isset( ... ) as language construct to check/prevent access to variables that aren't defined.

Reply:
Ah, I was already fixed one commit later https://github.com/SemanticMediaWiki/SemanticResultFormats/commit/dd6e74a3ffdee2cebe88f61a71480da3849876b0. Since I used the broken pull today I forgot to add it here, too. All in order for SRF.